src/rand.rs

```diff
@@ -432,3 +435,32 @@ mod fuchsia {
         fn zx_cprng_draw(buffer: *mut u8, length: usize);
     }
 }
+
+#[cfg(target_os = "vxworks")]
```

Comment:
I wonder why he doesn't use the getrandom crate? It is a pity we need to duplicate this.

Reply:
getrandom is kind of a new crate, the rust is keeping changing..